From 461f30670bcc5bacf9734847a0e24037edce90b8 Mon Sep 17 00:00:00 2001
From: Richard Hughes <richard@hughsie.com>
Date: Wed, 15 Feb 2023 14:22:53 +0000
Subject: [PATCH] Fix -Dbuild=library target

---
 meson.build | 36 ++++++++++++++++++------------------
 1 file changed, 18 insertions(+), 18 deletions(-)

diff --git a/meson.build b/meson.build
index 3eb58fba6..478de3126 100644
--- a/meson.build
+++ b/meson.build
@@ -590,25 +590,25 @@ if build_standalone
   ]
   subdir('plugins')
   subdir('src')
+  subdir('docs')
+
+  # append all the quirks into one big file and gzip it -- lzma is smaller, but needs libxmlb 0.3.3
+  builtin_quirk = custom_target('builtin-quirk',
+    input: plugin_quirks,
+    output: 'builtin.quirk',
+    capture: true,
+    command: ['cat', '@INPUT@'],
+  )
+  gzip = find_program('gzip')
+  custom_target('builtin-quirk-gz',
+    input: builtin_quirk,
+    output: 'builtin.quirk.gz',
+    capture: true,
+    command: [gzip, '-n', '-k', '--stdout', '@INPUT@'],
+    install: true,
+    install_dir: join_paths(datadir, 'fwupd', 'quirks.d'),
+  )
 endif
-subdir('docs')
-
-# append all the quirks into one big file and gzip it -- lzma is smaller, but needs libxmlb 0.3.3
-builtin_quirk = custom_target('builtin-quirk',
-  input: plugin_quirks,
-  output: 'builtin.quirk',
-  capture: true,
-  command: ['cat', '@INPUT@'],
-)
-gzip = find_program('gzip')
-custom_target('builtin-quirk-gz',
-  input: builtin_quirk,
-  output: 'builtin.quirk.gz',
-  capture: true,
-  command: [gzip, '-n', '-k', '--stdout', '@INPUT@'],
-  install: true,
-  install_dir: join_paths(datadir, 'fwupd', 'quirks.d'),
-)
 
 if build_daemon and libsystemd.found() and offline.allowed()
   install_symlink('fwupd-offline-update.service',
-- 
2.39.1

