From aa5c4d4af95ad159e32f73cdb8fc841ba87f5fe2 Mon Sep 17 00:00:00 2001
From: Richard Hughes <richard@hughsie.com>
Date: Tue, 21 Dec 2021 09:07:34 +0000
Subject: [PATCH] trivial: Fix compiling with -Dbuild=library

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index ca0476cbc..c9ffd8db6 100644
--- a/meson.build
+++ b/meson.build
@@ -193,8 +193,8 @@ endif
 if gio.version().version_compare ('>= 2.55.0')
   conf.set('HAVE_GIO_2_55_0', '1')
 endif
-if build_standalone
 gmodule = dependency('gmodule-2.0')
+if build_standalone
 if get_option('gudev')
   gudev = dependency('gudev-1.0', version : '>= 232')
   conf.set('HAVE_GUDEV', '1')
-- 
2.33.1

